PlaceRouteHierFlow/PnRDB/datatype.h

```diff
@@ -266,6 +287,7 @@ struct block {
   //int SBidx;
   //int counterpart;
   // Block pin
+  vector<PowerNet> PowerNets;
```

Comment:
@854768750 @Lastdayends Do we need this field (PowerNets in block) in the database? We have a similar one in hierNode.

Reply:
PowerNets in block does not contain physical information (rect, point). Now it only has connection information (which pins are connected to VDD or VSS). PowerNets in hierNode has physical information.